packages/field-label/package.json

```diff
@@ -65,7 +65,7 @@
         "@spectrum-web-components/shared": "1.4.0"
     },
     "devDependencies": {
-        "@spectrum-css/fieldlabel": "9.0.0-s2-foundations.16"
+        "@spectrum-css/fieldlabel": "9.0.1"
```

Comment:
Just out of curiosity, is there a reason we're not moving to the 10.1.0 version yet?

Reply:
Good question! I can see if the 10.1.0 version causes any other updates that might expand scope.